spec/datadog/core/crashtracking/component_spec.rb

```diff
@@ -0,0 +1,361 @@
+require 'spec_helper'
+require 'datadog/core/crashtracking/component'
+
+require 'webrick'
+require 'fiddle'
+
+RSpec.describe Datadog::Core::Crashtracking::Component,
+  skip: !CrashtrackingHelpers.supported? do
```

Comment:
Minor: Small suggestion -- consider turning this into a regular `before { skip (...) }`

1. Adding a new helper to spec_helper means *every* run of rspec, even when testing locally on completely different files will now have to load a helper that's only ever used by this one file

2. It's kinda weird that this whole file has one level of indentation extra just because of this linebreak.